Snakefile

```diff
@@ -447,6 +447,17 @@ rule tip_frequencies:
             --output {output.tip_frequencies_json}
         """
 
+def export_title(wildcards):
+    region = wildcards.region
+
+    if not region:
+        return "Genomic epidemiology of novel coronavirus"
+    elif region == "global":
```

Comment:
```suggestion
    elif region == "_global":
```
The region wildcard values come in with the attached underscores, so I think this will need to be like this - unless I'm missing somewhere where it's already been transformed so the `_` is gone?

Reply:
Ah, you are totally correct. I missed that the output of `_global` was "Global-focused subsampling" and not "Global subsampling".

Reply:
Repushed with your suggestion.